js/views/callinfoview.js

```diff
@@ -186,7 +186,7 @@
 		},
 
 		renderWhenInactive: function() {
-			if (this.ui.passwordInput.length === 0 || (OC._currentMenu && OC._currentMenu.hasClass('password-menu')) || this.ui.passwordInput.val() === '') {
+			if (!OC._currentMenu || !OC._currentMenu.hasClass('password-menu') || this.ui.passwordInput.length === 0 || this.ui.passwordInput.val() === '') {
```

Comment:
Nope, OC._currentMenu can be set to another menu like the apps menu or the setting one :)

Reply:
Exactly, but it is okay to reload the side bar when you have another menu open, just not the password field, because it would overwrite your changes.
Note this if is the evil thing, when we don't go inside, we will just retry later.

Reply:
Performance-side, why reload it on another menu click? Seems unnecessary :)

Reply:
It's not on click, it's on "model changed"  and that may be in the background when the room list is updated (happens on pull after a notification that there was a change...).
We jsut want to prevent rerendering while the user is typing a password.

Reply:
Ah okay! :)
Then it's :+1: for me!